tests/chainermn_tests/communicator_tests/test_communicator.py

```diff
@@ -281,6 +339,106 @@ def check_send_recv(param, use_gpu):
     check_send_and_recv_tuple(communicator, data)
 
 
+class SimpleMock(object):
```

Comment:
how about using `mock`

`mock` have been used in unittests for chainer and chainermn.
For example,
https://github.com/chainer/chainer/blob/ff238f9cacbea5dc1e8242f3442fc746e15df1d9/tests/chainer_tests/training_tests/extensions_tests/test_snapshot_writers.py#L3
https://github.com/chainer/chainer/blob/ff238f9cacbea5dc1e8242f3442fc746e15df1d9/tests/chainermn_tests/optimizer_tests/test_double_buffering_optimizer.py#L6

Reply:
Fixed!